toolkit/themes/shared/design-system/build/css/tokens-brand.css

```diff
@@ -13,11 +13,47 @@
 
 :root,
 :host(.anonymous-content-host) {
+  /* Base tokens */
+  /** Color **/
+  --color-canvas: light-dark(var(--color-white), var(--color-gray-90));
+
   /* Application tokens */
   /** Border **/
   --border-interactive-color: light-dark(var(--color-gray-60), var(--color-gray-50));
 
+  /** Color **/
+  --color-accent-primary: var(--brand-color-accent);
+  --color-accent-primary-active: var(--brand-color-accent-active);
+  --color-accent-primary-hover: var(--brand-color-accent-hover);
+
+  /** Font weight **/
+  --font-weight-light: 300;
+
+  /** Input **/
+  /*** Button ***/
+  --button-background-color: color-mix(in srgb, currentColor 7%, transparent);
+  --button-background-color-active: color-mix(in srgb, currentColor 21%, transparent);
+  --button-background-color-hover: color-mix(in srgb, currentColor 14%, transparent);
+
+  /** Link **/
+  --link-color: var(--brand-color-accent);
+  --link-color-active: var(--brand-color-accent-active);
+  --link-color-hover: var(--brand-color-accent-hover);
+  --link-color-visited: var(--link-color);
+
   /** Text **/
   --text-color: light-dark(var(--color-gray-100), var(--color-gray-05));
   --text-color-deemphasized: light-dark(color-mix(in srgb, currentColor 69%, transparent), color-mix(in srgb, currentColor 75%, transparent));
+
+  /* Unspecified */
+  --brand-color-accent: light-dark(var(--color-blue-50), var(--color-cyan-50));
+  --brand-color-accent-active: light-dark(var(--color-blue-70), var(--color-cyan-20));
+  --brand-color-accent-hover: light-dark(var(--color-blue-60), var(--color-cyan-30));
+  --font-size-large: 1.133rem;
+  --font-size-root: 15px;
+  --font-size-small: 0.867rem;
+  --font-size-xlarge: 1.467rem;
+  --font-size-xxlarge: 2.2rem;
+  --line-height: 1.5;
+  --line-height-small: 1.3;
```

Comment:
Do you mind adding these to the `TOKEN_SECTIONS` array so their sorted into the file?

It would be nice if we don't put anything in Unspecified. I left a comment on the test ticket to fail if we have tokens in here